### PR TITLE
HOCS-2642: add restart script

### PR DESCRIPTION
**Pull request by LiamMacP**

### CODE REVIEW

File: restart.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+#!/bin/bash
+set -euo pipefail
+
+export KUBE_NAMESPACE=${ENVIRONMENT}
+
+if [[ ${KUBE_NAMESPACE} == *prod ]]
+then
+    export KUBE_SERVER="https://kube-api-prod.prod.acp.homeoffice.gov.uk"
```
Comment (zuzak) [inline restart.sh:9]:
```suggestion
    export KUBE_SERVER="https://kube-api-prod.prod.acp.homeoffice.gov.uk"
    export KUBE_CERTIFICATE_AUTHORITY="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/acp-prod.crt"
```

Maybe we should gate this: `if (prod) quit()`.

File: restart.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+else
+    export KUBE_SERVER="https://kube-api-notprod.notprod.acp.homeoffice.gov.uk"
```
Comment (zuzak) [inline restart.sh:11]:
```suggestion
    export KUBE_SERVER="https://kube-api-notprod.notprod.acp.homeoffice.gov.uk"
    export KUBE_CERTIFICATE_AUTHORITY="https://raw.githubusercontent.com/UKHomeOffice/acp-ca/master/acp-notprod.crt"
```

File: restart.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+fi
+
+cd kd || exit 1
```
Comment (zuzak) [inline restart.sh:14]: We're not consuming any yaml so the working directory is irrelevant:
```suggestion
```

File: restart.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
+
+#!/bin/bash
```
Comment (zuzak) [inline restart.sh:2]:
```suggestion
#!/bin/bash
```

Make sure `chmod` is right too -- I can't tell just from the web UI.

File: restart.sh [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+set -euo pipefail
+
+export KUBE_NAMESPACE=${ENVIRONMENT}
+
+if [[ ${KUBE_NAMESPACE} == *prod ]]
+then
```
Comment (zuzak) [inline restart.sh:8]: This is more conventional (in our codebase, at least):
```suggestion
if [[ ${KUBE_NAMESPACE} == *prod ]] ; then
```

File: restart.sh (hunk 4 above)
Comment (LiamMacP) [reply restart.sh:2]: It has execution rights 👍🏼

File: restart.sh (hunk 1 above)
Comment (LiamMacP) [reply restart.sh:8]: I have added the certificate authority - wasn't sure it was required.

File: restart.sh (hunk 3 above)
Comment (LiamMacP) [reply restart.sh]: This has been changed 👍🏼

File: restart.sh (hunk 2 above)
Comment (LiamMacP) [reply restart.sh:11]: I have added the certificate authority - wasn't sure it was required.

File: restart.sh (hunk 5 above)
Comment (LiamMacP) [reply restart.sh]: I copied the format from the deploy - I have changed this as I tend to agree.

**Review by Paulcr-ho — APPROVED** [review]

**Pull request merged**